### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Sara Camassa David Berger

### DISCUSSION THREAD

**Discussion by Davtek11** [heig-vd-dai-course/heig-vd-dai-course#331] [Show and tell]
The objective of this project is to convert a text in a .txt file into a stylized version of this text (ex : simple ASCI art).

Link to the repository : https://github.com/Davtek11/DAI-Lab01

Final Commit Hash : 116ca45f47b870ab995cd70d2293d604894e2309

**Comment by GeraudSilvestri** [comment]
It seems a bit light. What would be the different PicoCLI parameters? The different stylized versions?

**Comment by Davtek11** [comment]
The parameters would be, for example, a font size, the type of ASCII art (if we have the time do the more than one), the characters the user wants to use, etc...

**Reply by ludelafo** [reply]
In this case, it is OK for me.

**Comment by Davtek11** [comment]
We forgot to put the link to the repository. Here it is : https://github.com/Davtek11/DAI-Lab01/tree/main
The final commit will come tomorrow !
